docs/userguide/pyproject_config.rst

```diff
@@ -70,6 +70,14 @@ The ``project`` table contains metadata fields as described by the
    # ... other project metadata fields as listed in:
    #     https://packaging.python.org/en/latest/guides/writing-pyproject-toml/
 
+.. important::
+   Support for
+   :doc:`project.license-files <PyPUG:specifications/pyproject-toml/#license-files>`
+   and SPDX license expressions in
+   :doc:`project.license <PyPUG:specifications/pyproject-toml/#license>` (:pep:`639`)
+   were introduced in version 71.0.0.
```

Comment:
Should this be `77.0.3` instead? Same for the other file.

Reply:
Umm, 77.0.0 introduces the features and really there was no code bugs in that version.
70.0.1 fixes docs/CI issues not related to the code itself, it is functionally identical to 77.0.1.
77.0.2 is another set of unrelated changes (I believe in distutils).
77.0.3 just postpones some of the removals (but those were not bugs to start with), but does not change functionality.

So if the users really buy into PEP 639 and are willing to handle the deprecations, anything >= 77.0.0 would work.

I am happy to change to 77.0.3, but toe people willing to adopt the bew standard, 77.0.0 is fully functional.

Reply:
I have to change this wrong 71 to 77 anyway...